content/en/docs/05/_index.md

```diff
@@ -125,7 +125,7 @@ spec:
               servicePort: 5000
 ```
 
-Apply the this Ingress definition using e.g. `kubectl create -f ingress.yml --namespace <namespace>`
+Apply this Ingress definition using, e.g., `kubectl create -f ingress.yml --namespace <namespace>`
```

Comment:
```suggestion
Apply this Ingress definition using e.g. `kubectl create -f ingress.yml --namespace <namespace>`
```

Reply:
The abbreviations i.e. and e.g. are always preceded by a punctuation mark. The question is, should a comma also follow the abbreviation? The answer is: It depends on whether you want to follow the American style or the British style.
https://jakubmarian.com/comma-after-i-e-and-e-g/

Reply:
I see, thanks for the reference. So let's use American English, as do probably all the other documentations.